src/providers/analytics.ts

```diff
@@ -69,4 +74,90 @@ export class Analytics {
       this.init().then(mixpanelIdentify);
     }
   }
+
+  /**
+    This exports the FS object from FullStory's tracking snippet
+  */
+  getFullStory(): any {
```

Comment:
this function is not needed because you added the script to the index.html so you have an access to a global variable for FullStory